scripts/commands/debugai.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- func: debugai
```

Comment:
Commands can also exist in modules, this is unusable if the module is turned off, but its in the main codebase - should be moved to modules